pkg/nodeps/utils_test.go

```diff
@@ -10,5 +10,27 @@ func TestRandomString(t *testing.T) {
 	randomString := RandomString(10)
 
 	// is RandomString as long as required
-	assert.Equal(t, 10, len(randomString))
+	asrt.Equal(t, 10, len(randomString))
+}
+
+// TestPathExplode tests PathExplode
+func TestPathExplode(t *testing.T) {
+	assert := asrt.New(t)
+
+	testSources := []string{
+		"sites/default/files",
+		"/sites/default/files",
+		"./sites/default/files",
+	}
```

Comment:
Ah, but where are the Windows examples :)

Reply:
Same as above, I expect this to work only on slashes, even in windows.. unless I understood things wrongly, but in the drupal.go you reference paths with slashes too.

Reply:
Every place we use file paths, we use the filepath stuff, it's important to be consistent about that. You can't just start randomly creating new ways to handle paths.  Did you want filepath.Split()? https://pkg.go.dev/path/filepath

Reply:
It's not doing the same as filepath.Split, it's a very specific thing I did (which might not be the bets) but it's connected to the way I am `filepath.Wallk`ing the dir on composer create command but then running all paths through a function that does the same  think as https://github.com/ddev/ddev/blob/0225d6f483adc606e9da1ffaf650c06d56f2882d/pkg/ddevapp/utils.go#L405. I made a function more general that does exactly the same thing.

I don't know how to change this withought being able to test it and it's not a simple change. If you have the time today, I'd like to walk you through what I implemented and why I think this will work the same on Windows as ToSlashes is being run and I only care about relative this. `PathRelativeSlashesExplode` could be a better name.

Reply:
This also confused me:

https://github.com/ddev/ddev/blob/0225d6f483adc606e9da1ffaf650c06d56f2882d/pkg/ddevapp/drupal.go#L501

it looks like that logics is supposed to work for windows and elsewhere, but it's only using slashes.

Reply:
That snippet is known to always run inside the container I believe, where it's always linux. You can probably tell I've had some bad experiences with bizarre results of not using filepath where it's required.

Reply:
Resolved as per our zoom chat